projects/packages/search/src/dashboard/components/donut-meter-container/index.jsx

```diff
@@ -19,22 +19,31 @@ const formatNumberWithSeparators = x => {
 /**
  * Returns a DonutMeterContainer describing resource usage.
  *
- * @param {object}prop - props to show usage info.
- * @param {number}prop.current - totalCount to the DonutMeter.
- * @param {number}prop.limit - segmentCount to the DonutMeter.
- * @param {string}prop.title - title to the DonutMeter.
- * @param {object}prop.tooltip - tooltip data
- * @returns {React.Component} DonutMeterContainer component.
+ * @param {object} props - Props
+ * @param {number} props.current - totalCount to the DonutMeter
+ * @param {number} props.limit - segmentCount to the DonutMeter
+ * @param {string} props.title - title to the DonutMeter
+ * @param {Function} props.iconClickedCallback - handler for click on "info" icon
+ * @param {Function} props.linkClickedCallback - handler for click on "details" link
+ * @param {object} props.tooltip - tooltip data
+ * @returns {React.Component} DonutMeterContainer component
  */
-const DonutMeterContainer = ( { current = 0, limit = 1, title, tooltip } ) => {
-	// TODO: Remove local callback in favour of props.
-	const tempCallback = () => {
-		// eslint-disable-next-line no-console
-		console.log( 'higher level callback...' );
-	};
-
-	const usageInfo =
-		formatNumberWithSeparators( current ) + '/' + formatNumberWithSeparators( limit );
+const DonutMeterContainer = ( {
+	current = 0,
+	limit = 1,
+	title,
+	tooltip,
+	iconClickedCallback,
+	linkClickedCallback,
+} ) => {
+	// Special case for "unlimited" requests, which has a limit of 64-bit PHP_INT_MAX.
+	const isUnlimitedRequests = limit > 1e18;
+	const localizedUnlimited = __( 'Unlimited', 'jetpack-search-pkg' );
+	const usageInfo = isUnlimitedRequests
+		? `0/${ localizedUnlimited }`
```

Comment:
I guess showing the current requests for the previous month wouldn't hurt and will give the user a sense of the new pricing. `0` doesn't really make a lot of sense here IMO.

Reply:
Good call. I'll have a look at this today.